tools/fluent/functions.md

```diff
@@ -0,0 +1,48 @@
+# Functions
+
+Functions can be used in Fluent to format data according to the current language’s rules, or can provide additional data to fine tune the translation.
+
+Fluent includes some [generic built-in functions](http://projectfluent.org/fluent/guide/functions.html#built-in-functions), like `NUMBER` and `DATETIME`, for number and date formatting, respectively. Such functions can be used in placeables and selectors. For example:
+
+```PROPERTIES
+dpi-ratio = Your DPI ratio is { NUMBER($ratio, minimumFractionDigits: 2) }
+
+your-score = { NUMBER($score, minimumFractionDigits: 1) ->
+        [0.0]   You scored zero points. What happened?
+       *[other] You scored { NUMBER($score, minimumFractionDigits: 1) } points.
+    }
+```
+
+This document describes additional functions available in Gecko products. Please refer to [this document](basic_syntax.md) for references about the terminology used.
+
+## PLATFORM
+
+`PLATFORM` is a function that allows localizers to taylor messages to the current platform. Allowed key variants are: `linux`, `win`, `macosx`, `android`, `other`.
```

Comment:
Typo: `to tailor`. And `Allowed variant keys` is perhaps better?

Reply:
The keys are still being discussed in https://bugzilla.mozilla.org/show_bug.cgi?id=1426063#c17.

Reply:
Yes, the plan is to leave this PR open until that bug is solved.